### DISCUSSION TITLE
xray手搓节点为何不支持旁路由使用？

### DISCUSSION THREAD

**Discussion by web-xietao** [XTLS/Xray-core#3891] [General]
xray server手搓节点，单机使用正常，在手机上，apple tv 等设备上分别用客户端连接使用正常。
但是局域网设备不可能都分别使用客户端连接，我一般局域网内是用apple tv 作为旁路由， 局域网内其他设备网关指向apple tv 上外网
但是xray手搓的节点，apple tv 本身使用正常可以上外网，但是局域网内的其他的设备指向它，外网是上不了的。。我用x-ui搭建的节点使用apple tv作为旁路由就一切正常。
是我哪里没有设置对对吗？

**Comment by tianlichunhong** [comment]
旁路路由方式是绝对可以的。只能说你的路由不正确，或者dns的问题。这属于低级问题，估计没有大佬回答你这个问题。
